db/payment_icons.yml

```diff
@@ -402,7 +402,11 @@
   name: vipps
   label: Vipps
   group: other
-  
+-  
   name: enets
   label: eNETS
   group: other
+  
```

Comment:
This line is missing a `-` to denote a new entry (see [example 2.4 from the YAML docs](https://yaml.org/spec/1.2/spec.html#id2760193))